src/omnicore/rules.cpp

```diff
@@ -301,6 +308,21 @@ bool IsAllowedOutputType(int whichType, int nBlock)
 }
 
 /**
+ * Adds a pending activation to the PendingActivations vector.
```

Comment:
It's currently unused, but I'm wondering, what you have in mind? Add support to lookup pending activations at runtime?